spec/abstractops.html

```diff
@@ -731,6 +731,11 @@ <h1>ISO 8601 grammar</h1>
           ASCIISign
           U+2212
 
+      UnpaddedHour :
+          DecimalDigit
+          `11`
+          `12`
```

Comment:
This isn't going to be sufficient to match `Etc/GMT-14`.

Reply:
right, oops

fixed

Reply:
A syntax-level bound to 14 doesn't seem right to me either (e.g., what if an Etc/GMT-15 time zone is introduced in the future?). I think I would instead expect a bound of 23, corresponding with allowed UTC offsets (and more similarity in the definitions of _UnpaddedHour_ and _Hour_).

Reply:
oops, quite right

fixed